.github/workflows/ts-tests.yml

```
@@ -26,9 +26,15 @@ jobs:
         run: |
           brew install coreutils
         if: ${{ runner.os == 'macOS' }}
+      - name: Clone reactor-ts
+        uses: actions/checkout@v3
+        with:
+          repository: lf-lang/reactor-ts
+          path: ./reactor-ts
+          ref: master
       - name: Perform TypeScript tests
         run: |
-          ./gradlew test --tests org.lflang.tests.runtime.TypeScriptTest.*
+          ./gradlew test --tests org.lflang.tests.runtime.TypeScriptTest.* -Druntime="./reactor-ts"
```

Why are we doing this? If you want to point to `master`, you can also write:
```suggestion
          ./gradlew test --tests org.lflang.tests.runtime.TypeScriptTest.* -Druntime="git://github.com/lf-lang/reactor-ts.git#master
```
(See https://github.com/lf-lang/lingua-franca/tree/master/test/TypeScript)

I am a little confused though that you found that `--external-runtime` didn't work, because I used it successfully earlier this week :thinking:

---

Earlier this week I mentioned that I was able to reproduce a bug using Gradle but not using `--external-runtime-path`, so I assumed that the bug fixed in this PR was the reason. `lfc` was silently failing to change the runtime in the generated `package.json` from `3.2.0` to a file system path; I confirmed this by locally running `lfc` before and after my changes.

Are you sure that `--external-runtime-path` was actually working, and not just silently doing nothing? If so, then there must be a recent change that is in `master` but not in `ts-upstream-delays` that fixed this problem.

---

The passing of command line options was broken in master and is fixed by https://github.com/lf-lang/lingua-franca/pull/1631

---

Thanks Christian! That makes the first change in this PR unnecessary.